keps/sig-node/5554-in-place-update-pod-resources-alongside-static-cpu-manager-policy/README.md

```diff
@@ -0,0 +1,1206 @@
+
+`CPUManagerPolicyOptions` allow to fine-tune the behavior of the `static` policy. The details of each option are described in their own KEP. As for kubernetes 1.33, the following options are available:
+
+* full-pcpus-only (GA, visible by default) (1.33 or higher) [KEP-2625](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2625-cpumanager-policies-thread-placement)
+* distribute-cpus-across-numa (beta, visible by default) (1.33 or higher) [KEP-2902](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2902-cpumanager-distribute-cpus-policy-option),
+* align-by-socket (alpha, hidden by default) (1.25 or higher) [KEP-3327](https://github.com/fromanirh/enhancements/blob/master/keps/sig-node/3327-align-by-socket/README.md)
+* distribute-cpus-across-cores (alpha, hidden by default) (1.31 or higher) [KEP-4176](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4176-cpumanager-spread-cpus-preferred-policy)
+* strict-cpu-reservation (beta, visible by default) (1.32 or higher) [KEP-4540](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/4540-strict-cpu-reservation/README.md)
+* prefer-align-cpus-by-uncorecache (alpha, hidden by default) (1.32 or higher) [KEP-4800](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4800-cpumanager-split-uncorecache) 
+
+Fine tuning of CPU Manager allows to combine above policy options, out of the 64 possible combinations, 40 combinations are today feasible given that there are two constraints:
+
+* full-pcpus-only is mutually exclusive with distribute-cpus-across-cores
+* distribute-cpus-across-cores is mutually exclusive with distribute-cpus-across-numa.
+
+The remaining options are independent and can be combined:
+
+* align-by-socket, 
+* strict-cpu-reservation, 
+* prefer-align-cpus-by-uncorecache 
+
+The Kubelet requires the total CPU reservation from `--kube-reserved` or `--system-reserved` and/or `--reserved-cpus` to be greater than zero when CPU management static policy is enabled. This KEP should ensure those reserved CPUs are kept during resize.
+
+### Kubelet interaction with Topology Manager
+
+In the below two sections we present sequence diagrams during Pod Admission and current behaviour upon an InPlacePodVertical Scaling attempt of a Guaranteed QoS PoD with static CPU Manager policy enabled ( Kubernetes v1.33 is used at the time of writing of this KEP )
+
+### Pod Admission Topology Manager interactions
+
+Topology Manager calculates the best fit from the topology hints retrieved from the CPU, Device and Memory Manager Hint providers, during the attempt of assignment of a PoD with topology Policy scheduled. In this context best means a fit which adheres with the configured combination of Topology, CPU, Device and Memory manager considering also kubelet CPU reservations and refers to NUMA affinity.
+
+![alt text](./pod_admission_topology_manager_interactions.png "Pod Admission Topology Manager Interactions")
```

Comment:
(Not a review comment. Thanks for creating this flow chart. It clearly illustrates the interaction between the topology manager and resource managers. Maybe worth adding this in the topology manager documentation, if it does not exist already.)